src/uiSelectDirective.js

```diff
@@ -411,7 +411,7 @@ uis.directive('uiSelect',
             }
 
             // Reset the position of the dropdown.
-            dropdown[0].style.opacity = 0;
+            dropdown[0].classList.add('ui-select-detached');
```

Comment:
Is it possible that this class is already being set before it is in this function?